bundles/org.eclipse.chronograph.swt/src/org/eclipse/chronograph/internal/swt/SWTChronograph.java

```diff
@@ -11,33 +11,33 @@
  *	Sergei Kovalchuk <sergei.kovalchuk@arsysop.ru> - 
  *												initial API and implementation
  *******************************************************************************/
-package org.eclipse.chronograph.viewer;
+package org.eclipse.chronograph.internal.swt;
 
 import java.time.LocalDate;
 import java.util.ArrayList;
 import java.util.List;
 
+import org.eclipse.chronograph.internal.api.Chronograph;
+import org.eclipse.chronograph.internal.api.Style;
 import org.eclipse.chronograph.internal.api.Styler;
 import org.eclipse.chronograph.internal.api.providers.ContainerProvider;
 import org.eclipse.chronograph.internal.base.UnitConverter;
-import org.eclipse.chronograph.internal.swt.BrickStyler;
-import org.eclipse.chronograph.internal.swt.GroupStyler;
-import org.eclipse.chronograph.internal.swt.RulerStyler;
-import org.eclipse.chronograph.internal.swt.SectionStyler;
-import org.eclipse.chronograph.internal.swt.StageStyler;
-import org.eclipse.chronograph.internal.swt.StatusStyler;
-import org.eclipse.chronograph.internal.swt.stage.ChronographStage;
-import org.eclipse.chronograph.swt.internal.stage.ChronographStageImpl;
+import org.eclipse.chronograph.internal.swt.stage.ChronographCanvas;
 import org.eclipse.swt.SWT;
 import org.eclipse.swt.layout.GridData;
 import org.eclipse.swt.widgets.Composite;
 
-public class ChronographViewer {
-	private final ChronographStage stage;
+/**
+ * 
+ * The main entry point to the SWT implementation of {@link Chronograph}
+ *
+ */
+public class SWTChronograph implements Chronograph {
```

Comment:
I assume SWTChronograph is not a suitable class name. The dependency fragment included in the package specification.